### PR TITLE
features(common): Latest Article Retrieval API [SEM-13]

### PR DESCRIPTION
**Pull request by Evert5n**
Latest Article Retrieval API [SEM-13] G1 display a list of updated articles with their title, abstract, publication date, and author. With GET

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:alpine
+WORKDIR /usr/app
+COPY ./package.json ./
+COPY ./package-lock.json ./
+RUN npm ci
+COPY . .
+RUN npm run build
+EXPOSE 3000
+CMD ["npm", "run", "start"]
```
Comment (Ditmar) [inline Dockerfile:9]: Nice I forget update the code

File: src/__mock__/mock-data.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const mockData = [
+    { title: "ARTICLE 1", author: "Author 1", summary: "Sumary 1", publicationDate: "2023-11-26T10:00:00.000Z" },
+    { title: "ARTICLE 2", author: "Author 2", summary: "Sumary 2", publicationDate: "2023-11-25T09:30:00.000Z" }
+  ];
+  
+export default mockData;
```
Comment (Ditmar) [inline src/__mock__/mock-data.ts:6]: beautiful

File: src/presentation/routers/user-router.ts [hunk 3]
```diff
@@ -13,6 +15,31 @@ const UserRouter = (db: NoSQLWrapper) => {
         const resultDb = await db.CreateUser(user);
         response.status(HttpStateCodes.OK).json({response: resultDb});
     })
+    router.delete('/user/:id', async(request, response) => {
+        const {id}=request.params;
+        try {
+            const result = await db.DeleteUsers(id);
+            result
+            ? response.status(HttpStateCodes.OK).send('Successfully deleted user with id ${id}')
```
Comment (Ditmar) [inline src/presentation/routers/user-router.ts]: nice job but is better if you send a json object please use json instead send

Comment (Ditmar) [inline src/presentation/routers/user-router.ts]: Another very important thing isn't to hardcode the messages it is better to use a central place to set the message I explain very good that in this pr.
check it -> https://github.com/Ditmar/api_sem_g2/pull/2/files
image

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice Job congratulations, please fix it some comments and this pr should be approved and merged

**Comment by Evert5n** [comment]
The latest articles retrieval API [SEM-13] G1 (Correction in user-router.ts change .send to .json) displays a list of updated articles with their title, abstract, publication date and author. With GET

**Review by Ditmar — APPROVED** [review]

**Pull request merged**